pytorch_lightning/profiler/pytorch.py

```diff
@@ -607,40 +628,26 @@ def __init__(
             self.context_names = {}
             self.running_stack = []
             self.profiler = None
-            self.lightning_module = None
+            self.lightning_module = None  # TODO: where is this set?
```

Comment:
See TODO

Reply:
It is set by the ProfilerConnector to enable recording on function names.

Reply:
```suggestion
            self.lightning_module = None  # set by ProfilerConnector
```